packages/runtime/test/lib/testUtils.ts

```diff
@@ -61,6 +61,45 @@ import {
 import { TestRuntimeServices } from "./RuntimeServiceProvider";
 import { TestNotificationItem } from "./TestNotificationItem";
 
+export async function expectThrowsAsync(method: Function | Promise<any>, customExceptionMatcher?: (e: Error) => void): Promise<void>;
+export async function expectThrowsAsync(method: Function | Promise<any>, errorMessagePatternOrRegexp: RegExp): Promise<void>;
+/**
+ *
+ * @param method The function which should throw the exception
+ * @param errorMessagePattern the pattern the error message should match (asterisks ('\*') are wildcards that correspond to '.\*' in regex)
+ */
+export async function expectThrowsAsync(method: Function | Promise<any>, errorMessagePattern: string): Promise<void>;
+
+export async function expectThrowsAsync(method: Function | Promise<any>, errorMessageRegexp: RegExp | string | ((e: Error) => void) | undefined): Promise<void> {
```

Comment:
we don't need this as it's already in the standard set of jest. see https://jestjs.io/docs/expect#rejects